src/auth-server/handlers/requestEmailConfirmationToken.ts

```diff
@@ -1,25 +1,31 @@
 import express from 'express'
-import { NotFoundError, TooManyRequestsError, BadRequestError } from '../errors'
-import { components } from '../generated/api-types'
-import { globalEm } from '../../utils/globalEm'
-import { Account, Token, TokenType } from '../../model'
 import { MoreThan } from 'typeorm'
+import { Account, EmailConfirmationToken, Session } from '../../model'
 import { ConfigVariable, config } from '../../utils/config'
-import { sendWelcomeEmail } from '../utils'
+import { globalEm } from '../../utils/globalEm'
+import { BadRequestError, TooManyRequestsError } from '../errors'
+import { components } from '../generated/api-types'
+import { sendLoginOrChangePasswordEmail, sendWelcomeEmail } from '../utils'
 
 type ReqParams = Record<string, string>
 type ResBody =
   | components['schemas']['GenericOkResponseData']
   | components['schemas']['GenericErrorResponseData']
 type ReqBody = components['schemas']['RequestTokenRequestData']
+type ResLocals = { authContext: Session }
 
 export const requestEmailConfirmationToken: (
```

Comment:
I dont know how, but I managed to generate 2 email tokens and orion now returns the first one, which turns out is `expired` as I try to make an account.

![image](https://github.com/Joystream/orion/assets/168457495/026fe0f6-0aff-45c8-b5c5-07527853c36f)